### DISCUSSION TITLE
Bluetooth control

### DISCUSSION THREAD

**Discussion by Lynwood** [rern/rAudio#470] [Feature Requests]
I started out searching for a controllable way to listen to my very large music collection in my car. My car has USB, but only allows 9,999 songs. I want to bluetooth it into my existing head unit, and control it thru the steering wheel via bluetooth controls.  All of this works perfectly when i use my phone. Pause, Play, change tracks, and music data is displayed. Unfortunately, my phone isnt big enough to hold all my music. :-(

Enter the Pi 4, I set it all up in the house, using Bose Bluetooth Earbuds. I was so happy when I got it to connect and actually play music. I moved it to the car, and it also connected and played, but there is no music data showing, and it will not switch tracks via bluetooth control.

This is the closest I've been to successfully achieving my goal.  Thanks so much for getting me this far!  I hope the missing items can be achieved eventually, as well.  :-)

Can anyone tell me, when using a DAC hat does it increase the audio fidelity over bluetooth?  Or just thru its outputs?  Thanks.  ;-)

**Comment by skiphansen** [comment]
Just it's outputs.

**Reply by Lynwood** [reply]
Thanks, I thought that was the case. Still, it sounds pretty darn good on every bluetooth device I've ever streamed music thru.. :-)
